### PR TITLE
Om gdm 222 geodesk url configurable

### PR DESCRIPTION
**Pull request by oliverm-be**
Sample here: 
http://dev.geomajas.org/geomajas-project-deskmanager-example-1.0.0-SNAPSHOT/manager/
http://dev.geomajas.org/geomajas-project-deskmanager-example-1.0.0-SNAPSHOT/desk/TEST_BE/

This pull request also contains GDM-215 example application branch, which is also available in pull request #34

My vote: Oliver May: +1 (binding)

### CODE REVIEW

**Comment by janvenstermans** [comment]
Jan Venstermans: +1

**Comment by jandm** [comment]
+1: binding

File: gwt/src/main/java/org/geomajas/plugin/deskmanager/client/gwt/common/util/GeodeskUrlUtil.java [hunk 1]
```diff
@@ -26,27 +23,24 @@
...
-		geodeskId = geodeskId.substring(geodeskId.indexOf(GdmLayout.geodeskPrefix) 
-				+ GdmLayout.geodeskPrefix.length()); 
-		geodeskId = geodeskId.substring(0, geodeskId.indexOf('/'));
-		
-		return geodeskId;
-		
+		return url;
 	}
 
+	static boolean isRelative(String url) {
```
Comment (jandm) [inline gwt/src/main/java/org/geomajas/plugin/deskmanager/client/gwt/common/util/GeodeskUrlUtil.java]: This does not look right !

File: gwt/src/main/java/org/geomajas/plugin/deskmanager/client/gwt/common/util/GeodeskUrlUtil.java [hunk 2]
```diff
@@ -26,27 +23,24 @@
...
+		return url.startsWith("http");
```
Comment (jandm) [inline gwt/src/main/java/org/geomajas/plugin/deskmanager/client/gwt/common/util/GeodeskUrlUtil.java]: Should be !startsWith()

**Comment by oliverm-be** [comment]
correct @jandm, I fixed it

**Pull request merged**